Symphony/Protocol/LPModelProtocol.h

```diff
@@ -10,7 +10,7 @@
 
 @protocol LPModelProtocol <NSObject>
 
-- (NSDictionary *) getClasMapping;
+- (NSDictionary *) getClassMapping;
```

Comment:
Can we change this name to `classMapping` instead of `getClassMapping` ? Objective-C convention. It's a preference, not a requirement